docs/trace/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,88 @@
+# Getting Started with OpenTelemetry .NET in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new webapp -o aspnetcoreapp
```

Comment:
lets do webapi so that the # of files to be checked in are minimal, and also gets rid of the other funny spellcheck thing you mentioned as well!

Reply:
Not opposed to that. FYI the reason I went with the razor web app style is when I searched "aspnetcore get started" I landed on [Tutorial: Get started with ASP.NET Core](https://learn.microsoft.com/aspnet/core/getting-started/?view=aspnetcore-7.0&tabs=windows). Since it does have a UI, I also added this:

![image](https://user-images.githubusercontent.com/28367120/212144740-bb9c5b39-3987-46ce-bd1b-bbb93efc6744.png)

Makes it kind of fun.

But agree it sucks to have all of those files in there. I'm going to mess with it a bit and see if they can be pulled at build time.